examples/passing/RecordUnionDupe.purs

```diff
@@ -0,0 +1,12 @@
+module Main where
+
+import Prelude
+import Data.Record (merge)
+import Control.Monad.Eff.Console (log, CONSOLE)
+import Control.Monad.Eff (Eff)
+import Unsafe.Coerce (unsafeCoerce)
+
+main :: Eff (console::CONSOLE) Unit
+main = do
+  let result = (merge {left:1} {left:"Done"}).left
+  log $ unsafeCoerce result
```

Comment:
It was returning the left side originally when the RowUnion was left biased however I think it makes more sense to be able to partially apply `merge` with the "defaultValues" and the other argument be the values which overwrite them, what do you think?